src/main/java/com/coinbase/prime/model/onchainaddressbook/NetworkType.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2025-present Coinbase Global, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.coinbase.prime.model.onchainaddressbook;
+
+public enum NetworkType {
```

Comment:
These aren't specific to address book. Add an onchain package?

Reply:
They're only used in the OnchainAddressBook, but I can move them to an enum package I guess?

Reply:
I moved to the `Common` package